packages/shared/src/components/plus/GiftPlusModal.tsx

```diff
@@ -195,27 +171,24 @@ export function GiftPlusModalComponent({
           </Typography>
           <PlusPlanExtraLabel
             color={PlusLabelColor.Success}
-            label={oneTimePayment.extraLabel}
+            label={giftOneYear?.extraLabel}
             typographyProps={{ color: TypographyColor.StatusSuccess }}
           />
           <Typography type={TypographyType.Body} className="ml-auto mr-1">
-            <strong className="mr-1">{oneTimePayment?.price}</strong>
-            {oneTimePayment?.currencyCode}
+            <strong className="mr-1">{giftOneYear?.price}</strong>
+            {giftOneYear?.currencyCode}
           </Typography>
         </div>
         <Typography type={TypographyType.Callout}>
-          Gift one year of daily.dev Plus for {oneTimePayment?.price}. Once the
+          Gift one year of daily.dev Plus for {giftOneYear?.price}. Once the
           payment is processed, they’ll be notified of your gift. This is a
           one-time purchase, not a recurring subscription.
         </Typography>
-        <Button
-          tag="a"
-          variant={ButtonVariant.Primary}
-          href={`${plusUrl}?giftToUserId=${selected?.id}`}
-          disabled={!selected}
-        >
-          Gift & Pay {oneTimePayment?.price}
-        </Button>
+        <Link href={`/plus/gift/${selected?.id}`} passHref>
```

Comment:
Should we maybe hide the userId from showing?

Reply:
We need the ID on the URL, not sure I understand.

Reply:
Yeah but can we somehow mask it maybe with NextJS?

Reply:
Not sure what it would do, it is pretty common for us to use IDs on URL params.